app/src/main/java/com/keylesspalace/tusky/AccountActivity.kt

```diff
@@ -237,13 +238,11 @@ class AccountActivity : BottomSheetActivity(), ActionButtonActivity, HasAndroidI
 
     private fun setupToolbar() {
         // set toolbar top margin according to system window insets
-        binding.accountCoordinatorLayout.setOnApplyWindowInsetsListener { _, insets ->
-            val top = insets.systemWindowInsetTop
-
-            val toolbarParams = binding.accountToolbar.layoutParams as CollapsingToolbarLayout.LayoutParams
+        ViewCompat.setOnApplyWindowInsetsListener(binding.accountCoordinatorLayout) { _, insets ->
```

Comment:
I thought they made extension functions instead of static methods for compat stuff now?

Reply:
Apparently only sometimes? I can't find them for these.